smartsim/_core/shell/shellLauncher.py

```diff
@@ -152,5 +114,63 @@ def impl(
             if run_command is not None
             else exe.as_program_arguments()
         )
+        return ShellLauncherCommand(
+            env, pathlib.Path(path), open(stdout_path), open(stderr_path), command_tuple
```

Comment:
It might be a good idea to add some ``# pylint: disable-next=consider-using-with`` comment(s) just so that when we do turn on `pylint` again it does not complain about these un-closed descriptors